src/graphql/queries/PoolSamples.mutation.gql

```diff
@@ -0,0 +1,12 @@
+mutation poolSamples($plate_id: Int!, $tag_set: Int!) {
+  poolSamples(
+    input: {
+      arguments: {
+        plateId: $plate_id
```

Comment:
Does this need to be plateBarcode rather than plateId? Don't know the answer but will be creating pools based on the barcode?